src/odemis/gui/util/img.py

```diff
@@ -1265,6 +1265,111 @@ def spectrum_to_export_data(spectrum, raw, unit, spectrum_range):
         return spec_plot
 
 
+def line_to_export_data(spectrum, raw, unit, spectrum_range):
+    """
+    Creates either raw or WYSIWYG representation for the spectrum line data
+
+    spectrum (model.DataArray): spectrum line data
+    raw (boolean): if True returns raw representation
+    unit (string): wavelength unit
+    spectrum_range (list of float): spectrum range
+
+    returns (model.DataArray)
+    """
+
+    if raw:
+        # attach distance as first column
```

Comment:
I think it's even clearer to pass to the exporter directly the metadata as is (ie, MD_WL_LIST if available, and MD_PIXEL_SIZE).
Up to the exporter to put the metadata at the right column or row, or in a special way.